src/main/io/displayport_max7456.h

```diff
@@ -20,7 +20,15 @@
 #include "config/parameter_group.h"
 #include "drivers/display.h"
 
-PG_DECLARE(displayPortProfile_t, displayPortProfileMax7456);
+typedef struct displayPortProfileMax7456_s {
```

Comment:
Good point.

The only part that is MAX7456 specific are the default values which are set to replicate the power on defaults.